src/components/AllReviews.jsx

```diff
@@ -39,6 +42,24 @@ const AllReview = () => {
     }
   };
 
+  const handleChange = (event) => {
+    setSearchParams((currentParams) => {
+      return { sort_by: event.target.value };
+    });
+  };
+  const handleChangeOrder = (event) => {
+    if (event.target.value === "asc") {
+      setAllReviews((currentReviews) => {
+        return [...currentReviews].reverse();
+      });
+    }
+    if (event.target.value === "desc") {
+      setAllReviews((currentReviews) => {
+        return [...currentReviews].reverse();
```

Comment:
This is certainly one way of changing the sort order, but your backend should hopefully have the capacity to do this in it's optional queries. It's more efficient for processes like this to be handled by the backend wherever possible